mathics/core/atoms.py

```diff
@@ -559,10 +559,10 @@ def __hash__(self):
         return self.hash
 
     def __str__(self) -> str:
-        return base64.b64encode(self.value).decode("utf8")
+        return f"<{len(self.value)}>"
```

Comment:
Not sure about that: __str__ should provide a str representation of the object. One possibility would be to replace line 566 by this one and keep the current behavior. @rocky, what do you think?

Reply:
Yes, you are correct that this is better. Will appear in next revision.

Reply:
Notice also that this form is just for the Output form:
In WMA, 
```
In[1]:= a=ByteArray[Table[Mod[i^2,255],{i,1,40}]]                               

Out[1]= ByteArray[<40>]


In[2]:= ToString[a]                                                             

Out[2]= ByteArray[<40>]

In[3]:=%//FullForm

Out[3]//FullForm= "ByteArray[<40>]"
```

However,

```
In[4]:= a//FullForm

Out[4]//FullForm= 
 
>   ByteArray["AQQJEBkkMUBRZHmQqcThASJFapG65RNCc6bbE0yHxARFiM0VXqn2Rg=="]

In[5]:= a//InputForm                                                            

Out[5]//InputForm= 
ByteArray["AQQJEBkkMUBRZHmQqcThASJFapG65RNCc6bbE0yHxARFiM0VXqn2Rg=="]

In[6]:= a//OutputForm                                                          

Out[6]//OutputForm= ByteArray[<40>]
``` 

So, the right thing to do is to implement this different behaviours in `atom_to_boxes`

Reply:
But it looks like there is second place that also needs to be changed for this.

Reply:
Fixed in a0c9e55f

Reply:
> 
> So, the right thing to do is to implement this different behaviours in `atom_to_boxes`

I don't like the way this is going..  The Form should be intercepting here not `atom_to_boxes()` testing on the Form. We went over that before. 

So let's leave this for now and address when we go over forms. I will, however make a note of this information.

Reply:
And probably the same thing is true for ToString.